src/Shapes/Sphere.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SphereProps, useSphere } from '@react-three/cannon';
+
+interface CustomSphere extends SphereProps{
```

Comment:
```suggestion
interface CustomSphere extends SphereProps {
```

These things should be fixed by using a linter when coding.